src/components/show-products/index.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { useHistory } from 'react-router';
+import Button from '../button';
+import { useProducts } from '../../hooks/useProducts';
+import ContentContainer from '../content-container';
+
+export const ShowProducts = () => {
+  const { products, loading } = useProducts();
+  const { push } = useHistory();
+
+  if (loading) {
+    return (
+      <ContentContainer>
+        <p>Loading...</p>
+      </ContentContainer>
+    );
+  }
+
+  if (products.length === 0) {
+    return (
+      <ContentContainer>
+        <p style={{ marginBottom: '20px' }}>
```

Comment:
Inline styles could be moved to styles files in order the keep the same structure in our code

```suggestion
        <p style={{ marginBottom: '20px' }}>
```

Reply:
I think this is the best way to maintain the structure. I already made the changes in this commit 19763a9, thanks for the feedback